.ci_helpers/run-test.py

```diff
@@ -27,13 +27,15 @@
     "root": {},  # This is to test the root folder.
     "convert": {},
     "calibrate": {},
+    "consolidate": {},
     "echodata": {},
+    "clean": {},
+    "mask": {},
+    "metrics": {},
+    "commongrid": {},
     "preprocess": {},
     "utils": {},
     "visualize": {},
```

Comment:
Here's the list in alphabetical order, leaving "root" as the first entry. I suggest we go with this order.

```suggestion
    "calibrate": {},
    "clean": {},
    "commongrid": {},
    "consolidate": {},
    "convert": {},
    "echodata": {},
    "mask": {},
    "metrics": {},
    "preprocess": {},
    "utils": {},
    "visualize": {},
```